src/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.Specific.cs

```diff
@@ -504,50 +503,6 @@ public void Windows_SetReadModeTo__PipeTransmissionModeByte()
             }
         }
 
-        [Fact]
```

Comment:
Both: UnixDomainSocketEndPoint is not available on UWP and NETFX, plus, it gets more organized and if we add abstract name support or want more tests in this area it will be clear.